ister.py

```diff
@@ -431,12 +439,12 @@ def copy_os(args, template, target_dir):
     cmd = "swupd verify --install"
     cmd += " --path={0}".format(target_dir)
     cmd += " --manifest={0}".format(template["Version"])
-    cmd += " --format={0}".format(args.format)
-    cmd += " --statedir={0}".format(args.statedir)
     if args.contenturl:
         cmd += " --contenturl={0}".format(args.contenturl)
     if args.versionurl:
         cmd += " --versionurl={0}".format(args.versionurl)
+    cmd += " --format={0}".format(args.format)
```

Comment:
Seems odd to move these two lines. Any reason?

Reply:
I split out my changes a little poorly across the commit, but the order was important to get the tests to pass because the command string changes otherwise